src/utils/Api.ts

```diff
@@ -98,3 +99,75 @@ export async function searchFolderLink(
     console.log(e);
   }
 }
+
+interface FormdataType {
+  userEmail: string;
+  userPw: string;
+}
+
+export async function loginRequest(formdata: FormdataType) {
+  try {
+    const response = await axios.post(`${BASE_URL}sign-in`, {
+      email: formdata.userEmail,
+      password: formdata.userPw,
+    });
+
+    if (response.status === 200) {
+      const res = response.data;
+      return res;
+    } else if (response.status === 400) {
+      console.log("없는 유저 정보");
+    } else if (response.status === 500) {
+      console.log("서버에러");
+    }
```

Comment:
r: 여기도 마찬가지로 진입 불가능한 코드가 있네요. axios에서 상태 코드 4~500번대이면 에러 발생해 catch절이 실행됩니다.

```suggestion
    const res = response.data;
    return res;
    // } else if (response.status === 400) {
    //   console.log("없는 유저 정보");
    // } else if (response.status === 500) {
    //  console.log("서버에러");
    // }
```